components/Form/Checkbox/__tests__/Checkbox.test.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```

Comment:
You can delete this file. We'll get plenty of coverage from Cypress + the RegistrationForm jest test.